language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/util/RegexpCompletionProvider.java

```diff
@@ -0,0 +1,688 @@
+                    RegexpCompletionProvider.RegexSnippet.DEF_RETURN.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_NEWLINE.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_TAB.get()));
+
+            // syntax chars
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_ASSERTION_START.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_ASSERTION_END.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_BACKSLASH_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_DOT_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_QUANTIFIER_ASTERISK_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_QUANTIFIER_PLUS_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_QUANTIFIER_QUESTION_MARK_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_LEFT_BRACE_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_RIGHT_BRACE_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_LEFT_SQUARE_BRACE_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_RIGHT_SQUARE_BRACE_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_LEFT_CURLY_BRACE_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_RIGHT_CURLY_BRACE_CHAR.get()));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_PIPE_CHAR.get()));
+
+            // unicode property chars
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_UNICODE_PROPERTY_CHAR.snippetBlock));
+            completionItems.add(new SnippetCompletionItem(ctx,
+                    RegexpCompletionProvider.RegexSnippet.DEF_NON_UNICODE_PROPERTY_CHAR.snippetBlock));
```

Comment:
Let's move this block into a separate method. It will improve readability of the code